cordova/build.sh

```diff
@@ -84,8 +84,14 @@ if [ ! -d $PROJECT ]; then
   cordova plugin add https://github.com/florentvaldelievre/virtualartifacts-webIntent.git
   checkOK
 
-  cordova plugin add https://github.com/phonegap/phonegap-plugin-barcodescanner.git
-  checkOK
+  if [ $CURRENT_OS != "WP8" ]
```

Comment:
ok, could you please add a 'echo "Using plugin XXXX for WINDOWS PHONE" so if windows phone release version 9 we can at least track the error?